discord/guild.py

```diff
@@ -3121,7 +3121,7 @@ async def bulk_ban(
         *users: Snowflake,
         delete_message_seconds: int | None = None,
         reason: str | None = None,
-    ) -> list[list[Snowflake], list[Snowflake]]:
+    ) -> tuple[list[Snowflake], list[Snowflake]]:
```

Comment:
Code below returns lists.

Reply:
Using that notation, it returns 2 lists in a tuple (which is what the new typehint is)

Reply:
Oops